server/catgenome/src/main/java/com/epam/catgenome/manager/gene/GffManager.java

```diff
@@ -1190,6 +1221,75 @@ private Block mergeExons(Block e1, Block e2) {
         return res;
     }
 
+    @SneakyThrows
+    public void genbankToGff(final String genebankFilePath, final Path gffFilePath) {
```

Comment:
Could you please extract logic related to this conversion to a separate class? `GffManager` is already too large.